src/frontend/python/mod.rs

```diff
@@ -1864,7 +1866,11 @@ fn super_circuit_halo2_mock_prover(rust_ids: &PyList, super_witness: &PyDict) {
         })
         .collect::<HashMap<u128, &str>>();
 
-    chiquito_super_circuit_halo2_mock_prover(uuids, super_witness)
+    chiquito_super_circuit_halo2_mock_prover(
+        uuids,
+        super_witness,
+        k.extract().expect("PyLong convertion failed."),
```

Comment:
conversion?

Reply:
Yes. It's type conversion between Python and Rust. PyLong is basically int.